ext/nnstreamer/tensor_filter/tensor_filter_tensorflow_lite.cc

```diff
@@ -46,6 +46,11 @@
 #include <tensorflow/contrib/lite/kernels/register.h>
 #endif
 
+#if defined(__ANDROID__) && (TFLITE_VERSION_MAJOR >= 2 || TFLITE_VERSION_MINOR >= 14)
```

Comment:
Q. Does tf-lite 1.14 support GPU delegate? 
If not, need to update the check condition.

At this stage, '__ANDROID__' definition is OK.
After your patch is merged, I will update the code to support both andorid and tizen.

Reply:
Thanks for pointing out! I've just checked with 1.15.2.
It does support gpu delegate, but its feature seems premature: support gl backend only, does not support quantized model, ... and the client code is quite different.

@jaeyun-jung and I decided support this feature only for 2.3.0, and I updated the code. Please check!